packages/block-editor/src/components/rich-text/editable.js

```diff
@@ -119,6 +119,18 @@ export default class Editable extends Component {
 			this.editorNode.className = classnames( nextProps.className, CLASS_NAME );
 		}
 
+		if ( this.props.start !== nextProps.start ) {
```

Comment:
@ellatrix, do you have a better idea how to tackle it? I don’t think it will scale well if we will have to maintain every possible attribute each tag can take.

Reply:
Maybe group them together in an object? `editableProps` or something like that?

Reply:
It doesn't have to be covered in this PR. However, we should plan to refactor it soon after and document it properly.

Reply:
Yeah, let's leave it as is for now.

Reply:
I actually think it would be good idea to start with grouping these four so adding/removing attributes in consistent and attribute agnostic.